docs/workflow/building/libraries/README.md

```diff
@@ -14,99 +21,96 @@ The build configurations are generally defaulted based on where you are building
 
 **Examples**
 - Building in release mode for platform x64 (restore and build are implicit here as no actions are passed in)
 ```
-libraries -c Release -arch x64
+./build.sh -subsetCategory libraries -c Release -arch x64
 ```
 
 - Building the src assemblies and build and run tests (running all tests takes a considerable amount of time!)
 ```
-libraries -restore -build -buildtests -test
+./build.sh -subsetCategory libraries -restore -build -buildtests -test
 ```
 
 - Building for different target frameworks (restore and build are implicit again as no action is passed in)
 ```
-libraries -framework netcoreapp
-libraries -framework netfx
+./build.sh -subsetCategory libraries  -framework netcoreapp
+./build.sh -subsetCategory libraries  -framework netfx
 ```
 
 - Build only managed components and skip the native build
 ```
-libraries /p:BuildNative=false
+./build.sh -subsetCategory libraries /p:BuildNative=false
 ```
 
 - Clean the entire solution
 ```
-libraries -clean
+./build.sh -subsetCategory libraries -clean
 ```
+
+For Windows, replace `./build.sh` with `build.cmd`.
+
 ### Build Native
-The native build produces shims over libc, openssl, gssapi, and libz.
-The build system uses CMake to generate Makefiles using clang.
-The build also uses git for generating some version information.
 
-The native component should be buildable on any system.
+The libraries build contains some native code. This includes shims over libc, openssl, gssapi, and zlib. The build system uses CMake to generate Makefiles using clang. The build also uses git for generating some version information.
 
 **Examples**
 
 - Building in debug mode for platform x64
 ```
-./src/libraries/Native/build-native debug x64
+./src/libraries/Native/build-native.sh debug x64
 ```
 
 - The following example shows how you would do an arm cross-compile build.
 ```
-./src/libraries/Native/build-native debug arm cross verbose
+./src/libraries/Native/build-native.sh debug arm cross verbose
 ```
 
-For more information about extra parameters take a look at the scripts `build-native` under src/Native.
+For Windows, replace `build-native.sh` with `build-native.cmd`.
 
 ### Building individual libraries
 
-**Note**: Before working on individual projects or test projects you **must** run `libraries` from the root once before beginning that work. It is also a good idea to run `libraries` whenever you pull a large set of unknown changes into your branch.
+**Note**: Recall that in order to work on individual library projects or run library tests it is necessary to have built CoreCLR (aka, the "runtime"). To do this you must run `build.sh` (or `build.cmd`) from the root once before beginning that work.
```

Comment:
Again - I would call out that you probably want a Release build of CoreCLR here. Don't just tell people to build from the root.